### DISCUSSION TITLE
ElasticSearch is mandatory for appbase.io Reactive components

### DISCUSSION THREAD

**Discussion by vmengh** [appbaseio/reactivesearch#1652] [Q&A]
Hi,
appbase.io provides a rich set of reactive components but is ElasticSearch backend  a mandate to support these or these reactive components work correctly with other backend data sources as well, by making tweaks to the API response and JSON format.

Has anyone tried it or come any closer to implementing it?
can this library components be used instead as we leverage Quasar or Vuetify vue components?

@siddharthlatest @lakhansamani  @bietkul @jyash97 

Thanks.

**Comment by siddharthlatest** [comment]
Hi @vmengh, we support `tranformRequest` and `transformResponse` properties in ReactiveBase (provider) component. You can use these to create an adaptor for a different backend.

We officially only support Elasticsearch, as long as one can map the declarative props (subset or fully) to another backend service, that can be used here.
